scripts/cardboardlinter

```diff
@@ -109,21 +123,17 @@ def load_config(config_file):
         Dictionary of the configurations where default values have been applied.
     """
     with open(config_file, 'r') as f:
-        config = yaml.load(f)
-
-    # find default configuration
-    default_keys = {key.replace('default_', ''): val for key, val in config.items()
-                    if 'default' in key}
-
-    # apply default configurations to all other keys that have not explicitly over write it
-    real_config = {key: val for key, val in config.items() if 'default' not in key}
-    # replace None
-    real_config = {key: {} if val is None else val for key, val in real_config.items()}
-    for def_key, def_val in default_keys.items():
-        for key, config in real_config.items():
-            config.setdefault(def_key, def_val)
-
-    return real_config
+        raw_config = yaml.load(f)
+
+    configs = []
+    for mapping in raw_config['linters']:
+        if len(mapping) != 1:
+            raise IOError('Every item in the linter section should only specify one linter.')
```

Comment:
Indeed. The error message may not be clear enough. I should add

    You probably have to add a dash in front of a linter name, e.g. `- import` instead of `import`.